### PR TITLE
Update ESRPCodeSigningTask to Version 5; Use WindowsAppSDK-EsrpCodeSigning-Steps.yml From WindowsAppSDKConfig Repository

### PR DESCRIPTION
**Pull request by kythant**
Cherrypick of https://github.com/microsoft/WindowsAppSDK/pull/4499

### CODE REVIEW

File: build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml [hunk 1]
```diff
@@ -193,33 +193,12 @@ stages:
...
 
     - ${{ if eq(parameters.SignOutput, 'true') }}:
-      - task: EsrpCodeSigning@2
-        inputs:
-          ConnectedServiceName: 'Project Reunion ESRP Code Signing Connection'
+      - template: AzurePipelinesTemplates/WindowsAppSDK-EsrpCodeSigning-Steps.yml@WindowsAppSDKConfig
+        parameters:
           FolderPath: $(build.artifactStagingDirectory)\FullNuget
           Pattern: 'Microsoft.WindowsAppSDK.Foundation*.nupkg'
+          KeyCode: 'CP-401405'
```
Comment (alexlamtest) [inline build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml:200]: Nonblocking - In other files, KeyCode is followed by displayName, is it intentional to drop displayName here? I know this won't affect functionality.

**Review by alexlamtest — APPROVED** [review]

File: build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml (hunk 1 above)
Comment (kythant) [reply build/AzurePipelinesTemplates/WindowsAppSDK-PackTransportPackage-Stage.yml:200]: Yea I will add a display name here. Not intentional

**Pull request merged**